<?xml version="1.0"?>
<launch>

    <param name="maxlen"        type="double" value="8.0" />    
    <param name="minlen"        type="double" value="0.0" />    
    <param name="max_ang_FOV"   type="double" value="4.3" />
    <param name="min_ang_FOV"   type="double" value="-0.6" />
    <param name="pcTopic"       type="str"    value="/velodyne_points" />
    <param name="imgTopic"      type="str"    value="/left_image" />
    <param name="imgTopic_blue"      type="str"    value="/right_image" />
    <param name="x_resolution"    type="double" value="0.25" />
    <param name="y_interpolation" type="double" value="10.0" />
    <param name="ang_Y_resolution"    type="double" value="2.05" />
    
    <group ns="matrix_file">
        <rosparam file="$(find lidar_camera_fusion)/cfg/cfg_params.yaml" />
    </group>
    
    <group ns="matrix_file_blue">
        <rosparam file="$(find lidar_camera_fusion)/cfg/cfg_params_blue.yaml" />
    </group>
    
    <node pkg="lidar_camera_fusion" type="lidar_camera_node" name="lidar_camera_node" output="screen"/>

<!-- rviz- -->
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find lidar_camera_fusion)/rviz/lidar_camera.rviz" />
  
</launch>
